src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -189,7 +189,7 @@ namespace JSONHelpers
 
     void FancyZonesData::SetActiveZoneSet(const std::wstring& uniqueID, const std::wstring& uuid /*TODO(stefan): deviceId instead of uniqueId in the future*/)
     {
-        if (!uuid.empty())
+        if (!uuid.empty() && deviceInfoMap.find(uniqueID) != deviceInfoMap.end())
```

Comment:
This won't fail ever as we add entry for current uniqueID in ZoneWindow->LoadSettings()->GetDeviceInfoFromTmpFile. But, let's keep it, no harm. We'll be safer in the future

Reply:
This won't fail, I agree, but do program needs these entries? Seems like it will add some unnecessary thing into container.

Reply:
It will add the entry for a monitor (device) first time editor is spawned for that particular monitor (device)